gradle.properties

```diff
@@ -0,0 +1 @@
+excludeIntegrationTests=netezza,sqlserver,teradata
```

Comment:
Is a new Chef recipe what you had in mind? I've just pushed something to create the file but I can't get to test this until later.